requirements.txt

```diff
@@ -1,10 +1,18 @@
 ckantoolkit
-Shapely>=1.2.13,<2.0.0
-pyproj==2.6.1
-OWSLib==0.18.0
 lxml>=2.3
 argparse
 pyparsing>=2.1.10
 requests>=1.1.0
 six
-geojson==2.5.0
\ No newline at end of file
+
+Shapely>=1.2.13,<2.0.0; python_version < '3.9'
```

Comment:
Do we need conditional installs for all these libraries? I can see how pyproj might require one, but I'd expect all the other ones to be able to use the same version in Python 3.8 and 3.9

Reply:
Updated here https://github.com/avdata99/ckanext-spatial/commit/bae9290395b252ee8e40056256fa694569d1d78b

We need cache for this tests. They take so long to run